docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -1113,6 +1113,12 @@ In that case, the items in the list are tried one by one until one succeeds.
 This behavior can be useful when working on a feature branch.
 For instance, you might want to align the config label with your branch but make it optional (in that case, use `spring.cloud.config.label=myfeature,develop`).
 
+=== Specifying Multiple Urls for the Config Server
```

Comment:
I would change this block of content to be the following:

=== Specifying Multiple URLs for Config Server

To ensure high availability when you have multiple instances of Config Server deployed and expect one or more instances to be unavailable from time to time, you can specify multiple URLs (as a comma-separated list under the `spring.cloud.config.uri` property). Note that doing so ensures high availability only when the Config Server is not running (that is, when the application has exited) or when a connection timeout has occurred. For example, if the Config Server returns a 500 (Internal Server Error) response or the Config Client receives a 401 from the Config Server (due to bad credentials or other causes), the Config Client does not try to fetch properties from other URLs. An error of that kind indicates a user issue rather than an availability problem. Such errors are often caused by problems with your security configuration on the client or server or YAML property files being in invalid format in your git repository. 

If you use HTTP basic security on your Config Server, it is currently possible to support per-Config Server auth credentials only if you embed the credentials in each URL you specify under the `spring.cloud.config.uri` property. If you use any other kind of security mechanism, you cannot (currently) support per-Config Server authentication and authorization.

Reply:
sure

Reply:
This would be more appropriate.

Such errors  can be caused by problems with your security configuration on the client or server or YAML property files being in invalid format in your git repository or any other reasons.

Reply:
Or if required we can just remove that line. It does not seem to be necessary in hindsight.